app/cafe.py

```diff
@@ -0,0 +1,23 @@
+from datetime import date
+
+from app.errors import (
+    NotVaccinatedError,
+    NotWearingMaskError,
+    OutdatedVaccineError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError("The visitor must have a vaccine")
+        if (visitor.get("vaccine").get("expiration_date") < date.today()
+                or visitor.get("vaccine").get("expiration_date") is None):
```

Comment:
This condition should come first because if expiration_date is None it will cause ValueError on previous line (you cannot compare Nonetype to datetime). Also you don't need to check that it `is None` explicitly